docs/docs/index.md

```diff
@@ -176,7 +176,8 @@ See `poetry help completions` for full details, but the gist is as simple as usi
 
 ```bash
 # Bash
-poetry completions bash > /etc/bash_completion.d/poetry.bash-completion
+mkdir -p ${BASH_COMPLETION_USER_DIR:-${XDG_DATA_HOME:-$HOME/.local/share}/bash-completion}/completions
+poetry completions bash > ${BASH_COMPLETION_USER_DIR:-${XDG_DATA_HOME:-$HOME/.local/share}/bash-completion}/completions/poetry
```

Comment:
```suggestion
BASH_COMPLETION_USER_DIR=${BASH_COMPLETION_USER_DIR:-${XDG_DATA_HOME:-$HOME/.local/share}/bash-completion}/completions
mkdir -p ${BASH_COMPLETION_USER_DIR}
poetry completions bash > ${BASH_COMPLETION_USER_DIR}/poetry
```

Reply:
Might also want to mention the use of `/etc` as a way to add to all users for a system-wide install.